### ISSUE TITLE
Doesn't set $PATH for OS X Yosemite install properly.

### ISSUE THREAD

**Issue by RecursiveLogic** [rust-lang-nursery/rustup.rs#758]
When I installed it, I had to manually write the path variable to `~/.bash_profile` in order to be able to use the commands properly rather than the default location that it writes it at.

**Comment by Phrohdoh** [comment]
This works as expected for me on macOS Sierra.
In fact `export PATH="$HOME/.cargo/bin:$PATH"` is the only line in my `~/.profile`.

@RecursiveLogic could you share your `$PATH`?

Mine is `/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/thill/.cargo/bin:/Users/thill/bin:/usr/local/sbin`

**Comment by RecursiveLogic** [comment]
Yeah I had to post `export PATH="$HOME/.cargo/bin:$PATH"` in my `~/.bash_profile` in order to get it to work. I'm running OS X Yosemite. Whenever I set my `$PATH` variables, I always do it in `~/.bash_profile`, nothing ever goes or works in `~/.profile`. I assumed it would be the same for at least a good portion of OS X users.

I have a lot of things in my `$PATH`, so it would be an ugly blob to post here.
